CONTRIBUTORS.yaml

```
@@ -1629,6 +1629,15 @@ mcmaniou:
     affiliations:
       - elixir-europe
 
+MD-Chem:
+    name: Mohua Das
+    joined: 2024-12
+    orcid: 0000-0002-8891-6333
+    email: dasmohua1991@gmail.com
+    affiliations: Genome Institute of Singapore
+    location: 
+        country: Singapore
```

Also use the country code here  
```suggestion
        country: SG
```